doc/source/whatsnew/v1.2.0.rst

```diff
@@ -484,7 +485,6 @@ Other
 - Fixed metadata propagation in the :class:`Series.dt` and :class:`Series.str` accessors (:issue:`28283`)
 - Bug in :meth:`Index.union` behaving differently depending on whether operand is a :class:`Index` or other list-like (:issue:`36384`)
 - Passing an array with 2 or more dimensions to the :class:`Series` constructor now raises the more specific ``ValueError``, from a bare ``Exception`` previously (:issue:`35744`)
-
```

Comment:
Can you add this back? Seems to be causing CI failure.

Reply:
Whoops, accidentally removed that line! Just added it back - hope this fixes the doc failure.